### PR TITLE
Feature sprint06 tijn new container model

### PR DESCRIPTION
**Pull request by tschmits**

### CODE REVIEW

File: delft3dworker/models.py [hunk 1]
```diff
@@ -436,6 +436,60 @@ def __unicode__(self):
...
+    which containers it requires, and in which states these containers are
+    desired to be.
+    """
+
+    scene = models.ForeignKey(Scene)
+
+    # delft3dgtmain.provisionedsettings
+    CONTAINER_TYPE_CHOICES = (
+        ('preprocess', 'preprocess'),
+        ('delft3d', 'delft3d'),
```
Comment (evetion) [inline delft3dworker/models.py:453]: simulation

File: delft3dworker/models.py [hunk 2]
```diff
@@ -436,6 +436,60 @@ def __unicode__(self):
...
+        ('non-existent', 'non-existent'),
+        ('created', 'created'),
+        ('restarting', 'restarting'),
+        ('running', 'running'),
+        ('paused', 'paused'),
+        ('exited', 'exited'),
+        ('dead', 'dead')
+    )
+
+    desired_state = models.CharField(
```
Comment (evetion) [inline delft3dworker/models.py]: non existent as default

File: delft3dworker/models.py [hunk 3]
```diff
@@ -436,6 +436,60 @@ def __unicode__(self):
...
+        max_length=16, choices=CONTAINER_STATE_CHOICES, blank=True)
+
+    docker_state = models.CharField(
```
Comment (evetion) [inline delft3dworker/models.py:476]: non existent as default

File: delft3dworker/models.py (hunk 1 above)
Comment (tschmits) [reply delft3dworker/models.py:453]: I sticked to the provisionedsettings convention (r30-35):

```
# delft3dworker
DELFT3D_IMAGE_NAME = 'delft3d'
POSTPROCESS_IMAGE_NAME = 'python27_container'
PREPROCESS_IMAGE_NAME = 'python27_container'
PROCESS_IMAGE_NAME = 'python27_container'
EXPORT_IMAGE_NAME = 'python27_container'
```

**Pull request merged**